frontend/package.json

```diff
@@ -263,7 +263,7 @@
     "moment-locales-webpack-plugin": "^1.2.0",
     "monaco-editor-core": "0.20.0",
     "monaco-editor-webpack-plugin": "^1.9.0",
-    "node-sass": "4.13.x",
+    "node-sass": "^4.14.1",
```

Comment:
Was this unintentional? (Good to update this, but we should do it as a separate PR.)

Reply:
This was intentional; It is needed for latest version of python on my system.  I will pull it out and place in a new PR.